scripts/delete-unauthorized-issues.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+REPO="ubiquity/devpool-directory"
+AUTHORIZED_ORG_IDS=(76412717 133917611 165700353)
+
+# Fetch issues with author login and author association
+issues=$(gh issue list --repo "$REPO" --limit 100 --json number,author,title)
+
+# Check if issues JSON is valid
+if [[ -z "$issues" || "$issues" == "[]" ]]; then
+    echo "No issues found or invalid JSON."
+    exit 0
+fi
+
+# Process each issue
+echo "$issues" | jq -c '.[]' | while read -r issue; do
+    issue_number=$(echo "$issue" | jq -r '.number')
+    issue_author_login=$(echo "$issue" | jq -r '.author.login')
+    issue_title=$(echo "$issue" | jq -r '.title')
+    author_association=$(echo "$issue" | jq -r '.authorAssociation')
+
+    if [[ ! " ${AUTHORIZED_ORG_IDS[@]} " =~ " ${issue_author_login} " && "$author_association" != "OWNER" ]]; then
```

Comment:
I assume this is only for bots. I suppose thats fine because humans shouldn't open issues here anyways.